.jenkins/Jenkinsfile-cryptography-wheel-builder

```diff
@@ -107,7 +103,18 @@ def build(version, label, imageName) {
                         virtualenv .venv -p ${pythonPath[version]}
                         source .venv/bin/activate
                         pip install -U wheel # upgrade wheel to latest before we use it to build the wheel
-                        CRYPTOGRAPHY_SUPPRESS_LINK_FLAGS="1" LDFLAGS="/usr/local/opt/openssl@1.1/lib/libcrypto.a /usr/local/opt/openssl@1.1/lib/libssl.a" CFLAGS="-I/usr/local/opt/openssl@1.1/include -mmacosx-version-min=10.9" pip wheel cryptography==$BUILD_VERSION --wheel-dir=wheelhouse --no-binary cryptography
+                        pip install cffi six idna asn1crypto ipaddress enum34
+                        pip download cryptography==$BUILD_VERSION --no-binary cryptography --no-deps
+                        tar zxf cryptography*
+                        cd cryptography*
+                        if [[ "${version}" == "py34" ]]; then
+                            PY_LIMITED_API='--py-limited-api=cp34'
```

Comment:
Is there some way we can have this dynamically handle `py3*` or something and use that ABI version?

Reply:
Such that if/when we update our wheels to be built against py35 and drop py3.4 support we don't have to update a bunch of `34` to `35`?

Reply:
Yeah, the idea is we'd only need to update in one place